src/filters/FilterSystem.ts

```diff
@@ -182,16 +182,15 @@ export class FilterSystem implements System
         }
         // get GLOBAL bounds of the item we are going to apply the filter to
 
-        const colorTextureSource = renderer.renderTarget.rootRenderTarget.colorTexture.source;
+        const colorTextureSource = renderer.renderTarget.renderTarget.colorTexture.source;
```

Comment:
I think we want to base inheriting based on the current render target not the root, right?

Reply:
Good point - I think technically we would want to take both into account?
- Like if anything in the filter stack does not have antialiasing, then if would be off?
- Find the lowest resolution in the filter stack? 

perhaps we keep track of that as we push / pop?

Reply:
I think there are use cases for `filter.antialias = 'on'` even if render does not use antialias. For example an interface layer with graphics needs antialiasing but the scene layer doesn't because it only contains sprite. And a filter within the interface layer with `filter.antialias = 'inherit'` should probably create an antialias filter texture, even though the renderer doesn't have antialiasing enabled.

Reply:
Thanks for explaining, thats a pretty cool use case!